core/ibft_test.go

```diff
@@ -1968,8 +1968,8 @@ func TestIBFT_ValidateProposal(t *testing.T) {
 				isProposerFn: func(_ []byte, _ uint64, _ uint64) bool {
 					return true
 				},
-				quorumFn: func(_ uint64) uint64 {
-					return quorum
+				hasQuorumFn: func(_ *proto.View, messages []*proto.Message) bool {
+					return len(messages) >= int(quorum)
```

Comment:
Abstracting this part to some helper function instead and use it whenever you need it in the tests (basically all the tests are using it in the same way)... WDYT?

```go
func(_ *proto.View, messages []*proto.Message) bool {
    return len(messages) >= int(quorum)
}
```

Reply:
Fixed in https://github.com/0xPolygon/go-ibft/pull/39/commits/00960bfa6ce2fe816db4088ed6662b28efc664d7